scripts/build/termux_step_create_pacman_package.sh

```diff
@@ -0,0 +1,95 @@
+termux_step_create_pacman_package() {
+	local TERMUX_PKG_INSTALLSIZE
+	TERMUX_PKG_INSTALLSIZE=$(du -s . | cut -f 1)
+
+	# From here on TERMUX_ARCH is set to "all" if TERMUX_PKG_PLATFORM_INDEPENDENT is set by the package
+	[ "$TERMUX_PKG_PLATFORM_INDEPENDENT" = "true" ] && TERMUX_ARCH=any
+
+	local PACMAN_FILE=$TERMUX_DEBDIR/${TERMUX_PKG_NAME}${DEBUG}_${TERMUX_PKG_FULLVERSION}_${TERMUX_ARCH}.pkg.tar.xz
+
+	local BUILD_DATE
+	BUILD_DATE=$(date +%s)
+
+	# Package metadata.
+	{
+		echo "pkgname = $TERMUX_PKG_NAME"
+		echo "pkgbase = $TERMUX_PKG_NAME"
+		if [ -n "$TERMUX_PKG_REVISION" ]; then
+			echo "pkgver = $TERMUX_PKG_VERSION-${TERMUX_PKG_REVISION}"
+		else
+			echo "pkgver = $TERMUX_PKG_VERSION-0"
+		fi
+		echo "pkgdesc = $(echo "$TERMUX_PKG_DESCRIPTION" | tr '\n' ' ')"
+		echo "url = $TERMUX_PKG_HOMEPAGE"
+		echo "builddate = $BUILD_DATE"
+		echo "packager = $TERMUX_PKG_MAINTAINER"
+		echo "size = $TERMUX_PKG_INSTALLSIZE"
+		echo "arch = $TERMUX_ARCH"
+		
+		local line
+		while read -r line; do
+			echo "license = $(sed 's/[ \t]*$//' <<< "$line")"
```

Comment:
Why is this sed necessary for all the fields? The read line seem to be without prefixed whitespace in all cases I can think of (but I guess I am missing something)

Reply:
Ok, check for trailing whitespace isn't really necessary there.